front/package.json

```diff
@@ -145,13 +153,18 @@
     "vite-tsconfig-paths": "^5.1.4",
     "vitest": "^3.0.4"
   },
+  "overrides": {
+    "eslint-plugin-storybook": {
+      "typescript": "~5.8"
+    }
+  },
   "scripts": {
     "compile": "tsc -b",
     "start": "vite",
     "start-container": "vite --host",
-    "build": "vite build",
-    "test": "vitest",
-    "lint": "eslint --cache --ext .ts,.tsx,.js,.jsx src scripts tests *.ts --max-warnings 0",
+    "build": "npm run build --workspaces --if-present && vite build",
```

Comment:
Do we need to build storybook here too ?

Reply:
Ah, good point, I didn't explain the motivation. I've added a new section "Should we build osrd-ui's storybook when building osrd's frontend?" in the PR description. Happy to change it if you prefer.